tests/epyccel/test_epyccel_mod.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from numpy.random import rand, randint, uniform
+from numpy import isclose
+
+from pyccel.decorators import types
+from pyccel.epyccel import epyccel
+from conftest import *
+
+def test_modulo_int_int(language):
+    @types(int, int)
+    def modulo_i_i(x, y):
+        return x % y
+
+    f = epyccel(modulo_i_i, language=language)
+    x = randint(1e6)
+    y = randint(1e6) + 1
```

Comment:
It's a bit strange to choose `x` and `y` this way. I would suggest to choose `y`  much smaller than `x`, so that the result of the modulo operation is less obvious.

By the way, can you explain why you're adding `1` to a random number?

Reply:
The `+1` is to avoid having `y=0`. `x % 0` results in a `ZeroDivisionError`